### PR TITLE
add lunar and kinetic release

### PR DESCRIPTION
**Pull request by k-okada**
https://github.com/ros/rosdistro/pull/15984#issuecomment-374680366

### CODE REVIEW

**Comment by mikaelarguedas** [comment]
Thanks!

Can you please add yourself and @7675t as maintainers in the package.xml files so that you receive emails from the buildfarm in case of failure ?

**Comment by k-okada** [comment]
@mikaelarguedas added me and @7675t as a maintainer.

File: tracks.yaml [hunk 1]
```diff
@@ -1,4 +1,27 @@
...
+    actions:
+    - bloom-export-upstream :{vcs_local_uri} :{vcs_type} --tag :{release_tag} --display-uri
+      :{vcs_uri} --name :{name} --output-dir :{archive_dir_path}
+    - git-bloom-import-upstream :{archive_path} :{patches} --release-version :{version}
+      --replace
+    - git-bloom-generate -y rosrelease :{ros_distro} --source upstream -i :{release_inc}
+    - git-bloom-generate -y rosdebian --prefix release/:{ros_distro} :{ros_distro}
+      -i :{release_inc} --os-name ubuntu
+#    - git-bloom-generate -y rosdebian --prefix release/:{ros_distro} :{ros_distro}
+#      -i :{release_inc} --os-name debian --os-not-required
```
Comment (mikaelarguedas) [inline tracks.yaml]: The same comment as https://github.com/ros-gbp/cartographer-release/pull/2#discussion_r177955439 applies here.

**Comment by mikaelarguedas** [comment]
Thanks @k-okada,

There is one last outstanding comment, once addressed this can be merged :+1:

**Pull request merged**